example/tsconfig.json

```diff
@@ -11,6 +11,9 @@
     "allowSyntheticDefaultImports": true,
     "resolveJsonModule": true,
     "esModuleInterop": true,
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "paths": {
+      "react-native-popable": ["../src"]
```

Comment:
So this is why all the warnings were showing up...! Great :)

Reply:
😇